assets/js/sidebar.js

```diff
@@ -83,14 +83,20 @@ export {breakpoint, closeSidebar, openSidebar, toggleSidebar, focusSearchInput}
 
 export function initialize () {
   setDefaultSidebarState()
+
   let lastWindowWidth = window.innerWidth
   $(window).resize(throttle(function () {
     if (lastWindowWidth !== window.innerWidth) {
       lastWindowWidth = window.innerWidth
       setDefaultSidebarState()
     }
   }, 100))
-  $('.sidebar-toggle').click(function () {
-    toggleSidebar()
+
+  $('.sidebar-toggle').click(toggleSidebar)
+
+  body.click((e) => {
```

Comment:
What do you think about - instead of listening to the `body.click` - using `$('.content').click(...)` ?

This way we won't have to check if sidebar/sidebar-toggle is one of the parents. :)

Reply:
Hi! Thank you for the feedback :) Good call – I considered it initially but incorrectly ruled it out thinking it would have issues. Seems to work perfectly. Force-pushed those changes.